drivers/gpio/gpio_mcux_rgpio.c

```diff
@@ -21,6 +21,9 @@
 	((const struct mcux_rgpio_config *)(_dev)->config)
 #define DEV_DATA(_dev) ((struct mcux_rgpio_data *)(_dev)->data)
 
+/* Default PAD config value for SCMI clock: Pull down, Slight Fast Slew Rate, X4 driver strength */
```

Comment:
`SCMI clock`?

Reply:
It should be SCMI pinctrl, has been fixed, thanks.